src/dog.sol

```diff
@@ -82,6 +88,8 @@ contract Dog is LibNote {
       address clip,
       uint256 id
```

Comment:
Wouldn't it be convenient to make the `id` indexed?

Reply:
And what about logging the `tab`? Is it not useful?

Reply:
Hmmm I'm actually seeing some collusions with the `Kick` event, which might be on purpose. And `tab` is in the `Kick` event so not actually necessary here.
I'm wondering if there was some analysis of which it is the best way to access all these values. In that way we could be sure we are adding the `indexed` keyword to the most correct values.

Reply:
I added `indexed` to the `id` in `Bark`, we are copying the convention in cat for `Bite` with `Bark`:
```
event Bark(
      bytes32 indexed ilk,
      address indexed urn,
      uint256 ink,
      uint256 art,
      uint256 tab,
      address clip,
      uint256 indexed id
    );
```
Where we pass in `due` in for `tab`: `emit Bark(ilk, urn, dink, dart, due, milk.clip, id);` which I personally find kind of confusing so I'm open to changing it.

Reply:
I remember we discussed that we should pass `due` instead of `tab` since `tab` is covered by `Kick`, but I think we should change it to `due` in the definition. (Or add `tab` as another param)

Reply:
Yeah I think calling it differently (`due` instead of `tab`) will help prevent confusion. There's already been some confusion with the present system (which logs a `tab` in both the Cat and Flipper, differing by the liquidation penalty).

Reply:
Looks good now.